HW_8/search_data.py

```diff
@@ -0,0 +1,16 @@
+from read_data import read_data
+from print_data import print_data
+
+
+def search_data(word, data):
+    if len(data) > 0:
+        lst = []
+        for item in data:
+            if word in item:
+                lst.append(item)
+        if lst == []:
+            return None
+        else:
+            return lst
+    else:
+        return None
```

Comment:
Если в функции не прописана конструкция `return`? она все равно будет возвращать `None`? Стало быть данную функцию можно переписать
```suggestion
def search_data(word, data):
    if len(data) > 0:
        lst = []
        for item in data:
            if word in item:
                lst.append(item)
        if lst:
            return lst

    return None
```